website/server/middlewares/cron.js

```diff
@@ -101,12 +102,18 @@ async function cronAsync (req, res) {
       return null;
     }
 
-    let _cronSignature = uuid();
+    // set _cronSignature to current time in ms since epoch time so we can make sure to wait at least an hour before trying again even if there is a timezoned
+    let _cronSignature = now.getTime();
+    // Calculate how long ago cron must have been attempted to try again
+    let timeoutThreshold = _cronSignature - CRON_TIMEOUT;
```

Comment:
Suggestion: rename variable to something more descriptive. Can't think of any suggestions right now.